server/config/middleware.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var userRoutes = require('../users/userRoutes.js');
+var messageRoutes = require('../messages/messageRoutes.js');
+var contactRoutes = require('../contacts/contactRoutes.js');
+
+module.exports = function (app, express) {
+
+  // instantiate express Routers for User, Message, and Contact 
+  // see: http://expressjs.com/guide/routing.html
+  var userRouter = express.Router();
+  var messageRouter = express.Router();
+  var contactRouter = express.Router();
+
```

Comment:
We need to serve up the main index.html in the client folder. Since there's no route for `/` that sends index.html back, you can add the following line of code here so that the server can serve up static assets from the client folder:

`app.use(express.static(__dirname + '/../../client'));`

That way, when going to `/`, it will serve up index.html.